src/Button/ButtonStyles.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components'
+
+const getWidth = (size: string) => {
+  switch (size) {
+    case 'big':
+      return '400px'
+    case 'normal':
+    default:
+      return '320px'
+  }
+}
+
+const getPadding = (size: string) => {
+  switch (size) {
+    case 'big':
+      return '55px 0'
+    case 'normal':
+    default:
+      return '22.5px 0'
+  }
+}
+
+export const ButtonWrapper = styled.button<any>`
+  outline: none;
+  cursor: pointer;
+  background: ${({ color }) => color || '#562878'};
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 3px;
+  box-shadow: 0 0 14px 1px rgba(0, 0, 0, 0.16);
+  width: ${({ size }) => getWidth(size)};
```

Comment:
We shouldn’t use fixed width for it. Just use padding Y according to design: 110 for big 45 for medium.